website/public.php

```diff
@@ -368,7 +368,8 @@ function calcFollowers($hero) {
+            '[{"offer":0,"id":185,"start":-1,"finish":-1,"curr":"UM","price":10000,"level":1,"promo":0,"notify":false}]',
```

Comment:
AM I understanding correctly, that tis removes the dev-heroes from the hero-shop?

Reply:
It is and indeed it shouldn't - didn't pay attention since i got them ! I'll also add an issue to define upcoming LTO.